core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1605,17 +1605,114 @@ secret_configs:
         namespace: test-credentials
         name: ci-pull-credentials-pack
         type: kubernetes.io/dockerconfigjson
-  # This secret stores the tokens of the SA which has permissions to push images to the CI central registry
-  # We will remove the one for api.ci after migrating the registry to app.ci
+  # This secret contains a docker config which has the permissions to pull the images from local registry
+  # and push the images to the ci central registry.
   - from:
-      token_image-pusher_api.ci_reg_auth_value.txt:
-        bw_item: build_farm
-        attachment: token_image-pusher_api.ci_reg_auth_value.txt
-      token_image-pusher_app.ci_reg_auth_value.txt:
-        bw_item: build_farm
-        attachment: token_image-pusher_app.ci_reg_auth_value.txt
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: build_farm
+          registry_url_bw_field: ocp3-registry-svc-lcal-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: ocp3-registry-svc-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: api.ci-registry-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: app.ci-registry-url
+          auth_bw_attachment: token_image-pusher_app.ci_reg_auth_value.txt
+    to:
+      - cluster: api.ci
+        namespace: ci
+        name: registry-push-credentials-ci-central
+        type: kubernetes.io/dockerconfigjson
+  - from:
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: build_farm
+          registry_url_bw_field: ocp4-registry-svc-lcal-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: ocp4-registry-svc-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: api.ci-registry-url
+          auth_bw_attachment: token_image-pusher_api.ci_reg_auth_value.txt
+        - bw_item: build_farm
+          registry_url_bw_field: app.ci-registry-url
+          auth_bw_attachment: token_image-pusher_app.ci_reg_auth_value.txt
     to:
-      - cluster_groups:
-        - build_farm
+      - cluster: app.ci
         namespace: ci
-        name: ci-central-registry-push-tokens
+        name: registry-push-credentials-ci-central
+        type: kubernetes.io/dockerconfigjson
+  - from:
+      .dockerconfigjson:
+        dockerconfigJSON:
+        - bw_item: build_farm
+          registry_url_bw_field: ocp4-registry-svc-lcal-url
```

Comment:
Isn't this `app.ci`? Why `ocp4`?

Reply:
Also you have a typo here: `ocp4-registry-svc-lcal-url`

Reply:
> Isn't this `app.ci`? Why `ocp4`?

This is a normal docker config for SA: It has the auth for `image-registry.openshift-image-registry.svc:5000`

```
oc --context app.ci get secret builder-dockercfg-hph8f -o yaml | yq -r '.data.".dockercfg"' | base64 -d | jq ". | keys[]"
"172.30.49.128:5000"
"image-registry.openshift-image-registry.svc.cluster.local:5000"
"image-registry.openshift-image-registry.svc:5000"
```

And for our `is/pipeline`:

```
  status:
    dockerImageRepository: image-registry.openshift-image-registry.svc:5000/ci-op-my2lbftt/stable
    publicDockerImageRepository: registry.build01.ci.openshift.org/ci-op-my2lbftt/stable
    tags:
    - items:
      - created: "2020-10-15T16:07:35Z"
        dockerImageReference: image-registry.openshift-image-registry.svc:5000/ci-op-my2lbftt/stable@sha256:f8b40076e049b18ed992b7e761a982e164a4f48c735879eb949702247eaa82af
```

We will use the pull secret for local image (dockerImageReference):
 image-registry.openshift-image-registry.svc:5000/ci-op-my2lbftt/stable@sha256:f8b40076e049b18ed992b7e761a982e164a4f48c735879eb949702247eaa82af

And we need a pull secret for ` image-registry.openshift-image-registry.svc:5000`.